src/misc.rs

```diff
@@ -43,13 +43,20 @@ impl LintPass for MiscPass {
                     // an enum is extended. So we only consider cases where a `_` wildcard is used
                     if arms[1].pats[0].node == PatWild(PatWildSingle) && 
                             arms[0].pats.len() == 1 {
+                        let body_code = snippet(cx, arms[0].body.span, "..");
+                        let suggestion = if let ExprBlock(_) = arms[0].body.node {
+                            body_code.into_owned()
+                        } else {
+                            format!("{{ {} }}", body_code)
```

Comment:
Can we instead manually create an exprblock and then convert to snipper here? It's a bit hard (use DUMMY_SP, and clone the original expr), but doable. This would make the output snippet have the correct indentation.

(Otherwise I'm fine merging this as-is)